.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+**Overview**
```

Comment:
This template is based on @hansl 's design template, stripped down to pull request size.